src/server/index.tsx

```diff
@@ -393,6 +393,13 @@ async function createSsrHtml(root: string, isoData: IsoDataOptionalSite) {
   </head>
 
   <body ${helmet.bodyAttributes.toString()}>
+    <a
+    href="#main-content"
+    class="skip-link"
+    onclick="document.getElementById('main-content').focus(); return false;"
```

Comment:
You don't have to make this function a string. You could make it:

```tsx
<a onClick={() => {
    document.getElementById('main-content')?.focus(); 
}}
>
    Jump to content
</a>
```

Reply:
Are you able to use JSX syntax here? This is in the `createSsrHtml` function which does not seem to be JSX but a HTML in a template string.

Reply:
That is a good point. I didn't realize this wasn't JSX in the small window github gave me.